product-catalog/test/productcatalog/services/CategoryServiceTest.java

```diff
@@ -0,0 +1,54 @@
+package productcatalog.services;
+
+import common.categories.CategoryUtils;
+import common.products.ProductUtils;
+import io.sphere.sdk.categories.Category;
+import io.sphere.sdk.categories.CategoryTree;
+import io.sphere.sdk.products.ProductProjection;
+import org.junit.Test;
+import productcatalog.exceptions.ProductNotFoundException;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CategoryServiceTest {
+
+    private final CategoryTree categories = CategoryTree.of(CategoryUtils.getQueryResult("categoryQueryResult.json").getResults());
+    private final List<ProductProjection> products = ProductUtils.getQueryResult("productProjectionQueryResult.json").getResults();
+    private final CategoryService categrieService = new CategoryService(categories);
+
+    @Test
+    public void getSiblingCategories() throws ProductNotFoundException {
+        final ProductProjection suttonBag = products.stream()
+                .filter(projection -> projection.getId().equals("254f1c0e-67bc-4aa6-992d-9a0fea1846b5"))
+                .findFirst().get();
```

Comment:
I would extract this method that searches for the product in both tests, since they way you obtain the product is not of interest for the test and it distracts from the test goal.